### PR TITLE
mysqli functions that always return true

### PR DESCRIPTION
**Pull request by Parsonswy**
Moved from #1413.

Reference;
[`mysqli_close`](https://github.com/php/php-src/blob/master/ext/mysqli/mysqli_api.c#L282)
[`mysqli_stmt_close`](https://github.com/php/php-src/blob/master/ext/mysqli/mysqli_api.c#L1480)
[`mysqli_field_seek`](https://github.com/php/php-src/blob/master/ext/mysqli/mysqli_api.c#L810)

### CODE REVIEW

**Comment by cmb69** [comment]
Hmm, prior to PHP 8.0.0, these functions emitted a warning and returned `null` or `false` on failure; as of PHP 8.0.0, in that case an `Error` is thrown. Not sure how to handle that in the docs; a `TypeError` would not be documented at all; a `ValueError` would, though. This seems more related to `ValueErrors` – at least the `false` return.

**Comment by Girgias** [comment]
Needs to be rebases, and a Changelog section added to describe the change in behaviour between PHP 7 and 8

File: reference/mysqli/mysqli_result/field-seek.xml [hunk 1]
```diff
@@ -53,7 +53,31 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   &return.success;
+   &return.true.always;
```
Comment (kamil-tekiela) [inline reference/mysqli/mysqli_result/field-seek.xml:56]: Strange, but in php-src it is still marked as return type bool

Comment (Girgias) [reply reference/mysqli/mysqli_result/field-seek.xml:56]: This should be fixed then, and have a changelog added, although the entity I created (``return.type.true``) for that is specific to 8.2.0... which was probably a bit of a mistake

Comment (Girgias) [reply reference/mysqli/mysqli_result/field-seek.xml:56]: Probably makes sense to actually add that changelog to the 2 other functions if it was enacted in 8.2

**Review by Girgias — APPROVED** [review]

File: reference/mysqli/mysqli_result/field-seek.xml (hunk 1 above)
Comment (kamil-tekiela) [reply reference/mysqli/mysqli_result/field-seek.xml:56]: The *value* has changed in 8.0, but the return *type* has changed in 8.2. I will follow up on the return type separately.

**Pull request merged**